package.json

```diff
@@ -11,7 +11,7 @@
     "cy:open": "yarn cypress open"
   },
   "engines": {
-    "node": "^12.20.0",
+    "node": ">=14",
```

Comment:
Com essa instrução (>=14) o heroku tentou instalar o node mais novo que ele tinha (Node.js v17.1.0) e aí quebrou umas coisas 

Failed to load ./.env.
Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:67:19)
    at Object.createHash (node:crypto:130:10)

Reply:
Acho que a gente pode optar por lockar uma versão então. Posso tentar a 16 e se não rolar deixamos na 14 por hora. O que acha?

Reply:
Beleza vamos tentar fazer o upgrade direto pro node 16, dai a gente vai de 14.

Nessa tentativa já pode resolver o conflito do lock?

Reply:
Claro, farei isso

Reply:
@miltonbsn Achas melhor fixar a versão ou deixar um range? `=14.x <=16.x` por exemplo

Reply:
vamos deixar fixo no 16 e testar